yt_dlp/extractor/fujitv.py

```diff
@@ -1,48 +1,52 @@
 # coding: utf-8
 from __future__ import unicode_literals
-
+from urllib.request import Request
 from .common import InfoExtractor
 
 
 class FujiTVFODPlus7IE(InfoExtractor):
-    _VALID_URL = r'https?://fod\.fujitv\.co\.jp/title/[0-9a-z]{4}/(?P<id>[0-9a-z]+)'
-    _BASE_URL = 'http://i.fod.fujitv.co.jp/'
-    _BITRATE_MAP = {
-        300: (320, 180),
-        800: (640, 360),
-        1200: (1280, 720),
-        2000: (1280, 720),
-        4000: (1920, 1080),
-    }
+    _VALID_URL = r'https?://fod\.fujitv\.co\.jp/title/(?P<sid>[0-9a-z]{4})/(?P<id>[0-9a-z]+)'
+    _BASE_URL = 'https://i.fod.fujitv.co.jp/'
 
     _TESTS = [{
-        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810075',
+        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40110076',
         'info_dict': {
-            'id': '5d40810075',
-            'title': '5d40810075',
+            'id': '5d40110076',
             'ext': 'mp4',
-            'format_id': '4000',
-            'thumbnail': 'http://i.fod.fujitv.co.jp/pc/image/wbtn/wbtn_5d40810075.jpg'
+            'title': '#1318 『まる子、まぼろしの洋館を見る』の巻',
+            'series': 'ちびまる子ちゃん',
+            'series_id': '5d40',
+            'description': 'md5:b3f51dbfdda162ac4f789e0ff4d65750',
+            'thumbnail': 'https://i.fod.fujitv.co.jp/img/program/5d40/episode/5d40110076_a.jpg',
         },
-        'skip': 'Expires after a week'
     }]
 
     def _real_extract(self, url):
-        video_id = self._match_id(url)
-        formats = self._extract_m3u8_formats(
-            self._BASE_URL + 'abr/tv_android/%s.m3u8' % video_id, video_id, 'mp4')
-        for f in formats:
-            wh = self._BITRATE_MAP.get(f.get('tbr'))
-            if wh:
-                f.update({
-                    'width': wh[0],
-                    'height': wh[1],
-                })
-        self._sort_formats(formats)
+        series_id, video_id = self._match_valid_url(url).groups()
+        self._download_webpage(Request(url, method='head'), video_id)
```

Comment:
```suggestion
        self._request_webpage(HEADRequest(url), video_id)
```
import `HEADRequest` from utils

Reply:
which utils? this?
![image](https://user-images.githubusercontent.com/40477634/151547452-dd34e1b1-7a42-4a4f-a96c-590cfe611271.png)